File: ws/santorum/u1/ws05CodingClases/ChikenFarmSimulator/src/ec_edu_espe_chickenfarmsimulator/model/Poop.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec_edu_espe_chickenfarmsimulator.model;



/**
 *
 * @author LABS-ESPE
 */
public class Poop {
        private int amount;

    public Poop(int amount){
        this.amount = amount;
    }

    public void setAmount(int amount) {
        this.amount = amount;
    }

    public int getAmount() {
        return amount;
    }

    @Override
    public String toString() {
        return "Poop{" + "amount=" + amount + '}';
    }
    



    }
    

File: hw/ramos/u1/hw07ChickenFarmSimulator/JavaScript/Poop.js
export default class Poop {
    #amount;

    constructor(amount) {
        this.#amount = amount;
    }

    getAmount() { return this.#amount; }
    setAmount(amount) { this.#amount = amount; }

    toString() {
        return `Poop{amount=${this.#amount}}`;
    }
}
